sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -786,7 +789,14 @@ static int sof_card_dai_links_create(struct device *dev,
 #endif
 
 	ssp_mask = SOF_SSP_GET_PORT(sof_rt711_rt1308_rt715_quirk);
-	ssp_num = hweight_long(ssp_mask);
+	/*
+	 * on generic TGL platform, I2S or SDW mode is supported
+	 * based on board rework. A acpi device is registered in
+	 * system only when I2S mode is supported, not SDW mode.
+	 * Here check acpi id to confirm I2S is supported.
```

Comment:
"ACPI," "ID" - also in the commit message. It looks a bit weird to me to see "I2S," "TGL," "SDW capitalised and "ACPI" and "ID" - not. BTW, there's again the question for "SDW" vs. "sdw." Also "An ACPI" both here and in the commit message

Reply:
thanks , I will unify the style